sys-auth/polkit/polkit-122.ebuild

```diff
@@ -145,7 +145,7 @@ src_install() {
 	fi
 
 	diropts -m 0700 -o polkitd
-	keepdir /usr/share/polkit-1/rules.d
+	use prefix || keepdir /usr/share/polkit-1/rules.d
```

Comment:
We should just drop this entirely. The keepdir is pointless because polkit installs 50-default.rules there anyway.